src/client/components/ContractsFestivals.js

```diff
@@ -1,43 +1,71 @@
 import PropTypes from 'prop-types';
 import React, { useState, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
+import DateTimePicker from 'react-datetime-picker';
 
+import translate from '~/common/services/i18n';
 import ButtonOutline from '~/client/components/ButtonOutline';
 import EthereumContainer from '~/client/components/EthereumContainer';
 import { addPendingTransaction } from '~/client/store/ethereum/actions';
 import {
   isFestivalInitialized,
   initializeFestival,
   TX_INITIALIZE_FESTIVAL,
+  isFestivalDeactivated,
+  deactivateFestival,
+  TX_DEACTIVATE_FESTIVAL,
 } from '~/common/services/contracts/festivals';
 import {
   usePendingTransaction,
   useOwnerAddress,
 } from '~/client/hooks/ethereum';
 
-const ContractsFestivals = ({ chainId }) => {
+const InitializeFestival = ({ chainId, owner }) => {
```

Comment:
Small naming thing: All sub components in one file start with the same word, so it could be `ContractsFestivalsInitialize` for example or `ContractsFestivalsDeactivate`. Also the sub components come after the main one so you see the most important one directly when opening the file.